CHANGELOG.md

```diff
@@ -1,3 +1,221 @@
+## v0.5.0 - 2020-10-23
+
+We're happy to announce the release of Lokomotive v0.5.0(Eurostar).
+
+This release packs new features, bug fixes, code optimizations, platform updates, security hardening, and much more.
+
+### Changes in v0.5.0
+
+#### Kubernetes updates
+
+- Update Kubernetes version to
+  [`v1.19.3`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#v1193)
+  ([#1030](https://github.com/kinvolk/lokomotive/pull/1030)).
+
+#### Platform updates
+
+##### AKS
+
+- Update Kubernetes version to `1.18.8`
+  ([#1071](https://github.com/kinvolk/lokomotive/pull/1071)).
+
+##### Baremetal
+
+- Expose CNI MTU on the baremetal platform
+  ([#977](https://github.com/kinvolk/lokomotive/pull/977)).
+
+#### New components
+
+- Add component `web-ui`
+  ([#981](https://github.com/kinvolk/lokomotive/pull/981)),
+  ([#1100](https://github.com/kinvolk/lokomotive/pull/1100)).
+- Add component `inspektor-gadget`
+  ([#1076](https://github.com/kinvolk/lokomotive/pull/1076)).
+
+#### Component updates
+
+- Update Velero component for Packet [OpenEBS and restic plugin support]
```

Comment:
Link here is broken.
![Selection_264](https://user-images.githubusercontent.com/16539896/96994327-da4a9f80-152c-11eb-9fa7-5bd52d336d97.png)

Reply:
It works for me. Anyone else has this problem?

Reply:
works for me as well.

Reply:
That bracket is some for of comment

Reply:
Why not `()` then?